### ISSUE TITLE
Transcode results in blank output in specific key(s)

### ISSUE THREAD

**Issue by jonpikum** [ChordPro/chordpro#334]
## Describe the bug
It seems to be the case that when using `--transcode=XXX` in specific keys, some notes are blank. 

### To Reproduce
run `chordpro -X --transcode=roman test.cho -o testOut.cho`

### Expected behavior
Transcoded notes should be rendered for each note in both C and G.

### Attachments
```chordpro
{title:Test Song}
{key:G}

{sob:Key of C}
[C]C [Dm]D [Em]E [F]F [G]G [Am]A [Bm]B
[F/C]F/C [F2/C]F2/C
{eob}

{sob:Key of G}
[G]G [Am]A [Bm]B [C]C [D]D [Em]E [F#m]F#
[C/G]C/G] [C2/G]C2/G
{eob}
```

<details><summary>C Roman Output (Correct)</summary>
<p>

```cho
{title: Test Song}
{key: I}

{start_of_bridge: Key of C}
[I]C [ii]D [iii]E [IV]F [V]G [vi]A [vii]B
[IV/I]F/C [IV2/I]F2/C
[IV][*/][I]F/C [IV2][*/][I]F2/C
{end_of_bridge}

{start_of_bridge: Key of G}
[V]G [vi]A [vii]B [I]C [II]D [iii]E [#iv]F#
[I/V]C/G [I2/V]C2/G
[I][*/][V]C/G [I2][*/][V]C2/G
{end_of_bridge}
```

![image](https://github.com/ChordPro/chordpro/assets/1560936/a8dd3d37-289b-45d9-a7b6-b9e83dda6e13)

</p>
</details>

<details><summary>G Roman Output (Incorrect)</summary>
<p>

```cho
{title: Test Song}
{key: V}

{start_of_bridge: Key of C}
[IV]C [v]D [vi]E [#VI]F []G []A []B
[#VI/IV]F/C [#VI2/IV]F2/C
[#VI][*/][IV]F/C [#VI2][*/][IV]F2/C
{end_of_bridge}

{start_of_bridge: Key of G}
G []A []B [IV]C [V]D [vi]E [vii]F#
[IV/I]C/G [IV2/I]C2/G
[IV][*/][I]C/G [IV2][*/][I]C2/G
{end_of_bridge}
```

![image](https://github.com/ChordPro/chordpro/assets/1560936/c6ddf49d-ec50-40db-a74b-a71eb54b008a)

</p>
</details>

<details><summary>G Nashville Output (Incorrect - See Inversions)</summary>
<p>

```cho
{title: Test Song}
{key: 5}

{start_of_bridge: Key of C}
[4]C [5m]D [6m]E [#6]F [1]G [2m]A [3m]B
[#6/4]F/C [#62/4]F2/C
[#6][*/][4]F/C [#62][*/][4]F2/C
{end_of_bridge}

{start_of_bridge: Key of G}
[1]G [2m]A [3m]B [4]C [5]D [6m]E [7m]F#
[4]C/G [42]C2/G
[4][*/][]C/G [42][*/][]C2/G
{end_of_bridge}
```

![image](https://github.com/ChordPro/chordpro/assets/1560936/9f8690f8-22e5-4fee-923b-96e2742a67e7)

</p>
</details> 

Please note that when using the key of G the same chords are *also* missing from the `.txt`, and `.html` output formats.
Screenshots are of the PDF output.

## System information
 - OS: Void Linux, Kernel `6.6.7`
 - Where did you get ChordPro from? CPAN without root
 - ChordPro version: `6.030`
 - Perl version: `v5.38.2`

Let me know if I can provide any more details.
It seems some of the other transcodes are okay? Didn't see the issue when using `latin` for example, but I didn't test them all.

**Comment by sciurius** [comment]
Fixed in 6.030_064.

**Comment by jonpikum** [comment]
Confirmed fixed using ChordPro `6.040` from cpan.
